src/pattern/application/pattern.service.ts

```
@@ -10,64 +10,81 @@ import { SambaNovaService } from '~samba-nova/application/samba-nova.service';
-        response = await this.openrouterService.generateText(param.pattern);
-        break;
-      case PROVIDER.SAMBA_NOVA:
-        response = await this.sambaNovaService.generateText(param.pattern);
-        break;
-      default:
-        response = await this.geminiService.generateText(param.pattern);
-    }
+  async decodeToCROML(param: {
+    pattern: string;
+    provider?: PROVIDER;
+  }): Promise<Object> {
```

Biome

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)